### DISCUSSION TITLE
swift-composable-architecture dependency version need updated. this repo request the version is lower than 1.0.0

### DISCUSSION THREAD

**Discussion by StormXX** [pointfreeco/composable-core-location#44] [General]
Failed to resolve dependencies Dependencies could not be resolved because root depends on 'composable-core-location' 0.3.0.
'composable-core-location' 0.3.0 cannot be used because 'composable-core-location' 0.3.0 depends on 'swift-composable-architecture' 0.43.0..<1.0.0 and root depends on 'swift-composable-architecture' 1.14.0.

**Comment by StormXX** [comment]
I found a PR 
https://github.com/pointfreeco/composable-core-location/pull/29
When will the repo release a new version for TCA 1.x?

**Comment by mbrandonw** [comment]
Hi @StormXX, this repo is not a top priority right now. We think it would be best for people to create their own location clients. It doesn't take much work to do, and it'd be better in the long term for you to have full control over the code rather than depending on a library for it. I am going to convert this to a discussion for now, but feel free to continue the conversation over there.
